src/Pact/Types/Swagger.hs

```diff
@@ -69,6 +96,36 @@ fixedLength i =
   set maxLength (Just $ fromIntegral i) .
   set minLength (Just $ fromIntegral i)
 
+replaceOrModify :: (Eq a) => (a, a) -> (a -> a) -> a -> a
```

Comment:
It's unclear why this is needed, as it seems like two unrelated cases -- wouldn't we know if we were replacing vs modifying? If it is genuinely necessary, please provide a haddock that gives the motivation/requirement.